FE/src/index.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```

Comment:
For the font-family, I think we will eventually have to align with the practices currently adopted at [https://github.com/jenkins-infra/jenkins.io](https://github.com/jenkins-infra/jenkins.io). But we can do that in a follow-up PR.